### ISSUE TITLE
Support setting log time in logging view

### ISSUE THREAD

**Issue by Lineflyer** [cgeo/cgeo#12991]
### Tell us your idea!

A user contacted us via support with the problem, that he sometimes accidentally forgets to offline log a certain cache while out caching. If he later on adds the offline log for the forgotten cache, the c:geo history list and the field note file (timestamp in file) reflects a "wrong" sequence of logs.

He would like to have the ability to manually correct the sequence.

### What solution would you suggest?

The user would like to have the possibility to explicitly set the time (in addition to the date) for his log in the logging view. It should be prefilled with current time while composing the log.
But (in case of offline logs) it would then be possible to re-access the offline stores log and change the time as needed before exporting his field notes / sending the logs.

### Is there an alternative?

No change.

### Additional Information

Out of curiosity: 
Do we only use the timestamp for field note export or is the time also part of the information we send to the server for an online log?

**Comment by ztNFny** [comment]
> Do we only use the timestamp for field note export or is the time also part of the information we send to the server for an online log?

Depends on the connector. For GC.com the date is sent as yyyy-MM-dd, so without time.

**Comment by Lineflyer** [comment]
> Depends on the connector. For GC.com the date is sent as yyyy-MM-dd, so without time.

So this means it would be a pure field note feature (at least for gc.com).
The question would be (if after all this feature request is accepted) if this time field should be prominent in the logging view?

Is there some kind of combined date/time picker, so that the logging view shows the date, but when tapping on the date a combined date/time picker is shown. This would be less intrusive and helpful for field note and non-GC loggers.

**Comment by ztNFny** [comment]
Can't the scenario where log time is needed more easily be solved by inserting a text with time in log text?

**Comment by eddiemuc** [comment]
c:geo already provides the possibility to log a time together with a date. This is used for trackables for platforms supporting this. It would be fairly easy to enable this also for cache logs and/or other services:

![image](https://user-images.githubusercontent.com/6909759/168004946-699bc6b5-8715-4ab2-8cb1-59236dd0b6f4.png)

**Comment by Lineflyer** [comment]
Yes, that would probably be the easiest option and also fine for me.

My statement about some combined field/picker  was just taking into consideration
- using less screen space
- making it less intrusive to users not needing to set a time for their logs 

But I won't insist on additional effort while there is an easy reusable option.
